3p/iframe-messaging-client.js

```diff
@@ -100,6 +100,19 @@ export class IframeMessagingClient {
    */
   setupEventListener_() {
     listen(this.win_, 'message', event => {
+      try {
+        if (this.hostWindow_ && !this.hostWindow_.postMessage &&
```

Comment:
If this.hostWindow_ is the window object of a non-friendly frame, then accessing this.hostWindow_.anything is prohibited.
Hence the check for this.hostWindow_.postMessage - if that exists, then this.hostWindow_ is not an iframe element. It's a window (though perhaps the window of an iframe), and likely to be unfriendly.
But if we get past the !this.hostWindow_.postMessage then we know it's a frame. I agree the closure typing makes the tag name check unnecessary and will remove.